Sources/MongoSwift/MongoClient.swift

```diff
@@ -191,8 +191,7 @@ public class MongoClient {
 
     private let operationExecutor: OperationExecutor
 
-    /// Default size for a client's NIOThreadPool.
-    // TODO: SWIFT-705 document size justification.
+    /// Default size for a client's NIOThreadPool. TODO: SWIFT-705 document size justification.
```

Comment:
we probably want to move the todo to the line above the docstring, otherwise it'll be rendered by jazzy.

Reply:
I think we will remove this before the next release occurs, but I've moved it anyway in case not 👍